pkg/storkctl/clusterpair.go

```diff
@@ -936,3 +954,29 @@ func createNamespace(config string, namespace string) error {
 	}
 	return nil
 }
+
+func verifyBackupLocationExistence(config string, blName string, namespace string) error {
+	conf, err := getConfig(config).ClientConfig()
+	if err != nil {
+		return err
+	}
+	storkOps, err := storkops.NewForConfig(conf)
+	if err != nil {
+		return err
+	}
+	bl, err := storkOps.GetBackupLocation(blName, namespace)
+	if err != nil {
+		return err
+	}
+	if len(bl.Location.SecretConfig) > 0 {
+		coreOps, err := core.NewForConfig(conf)
+		if err != nil {
+			return err
+		}
+		_, err = coreOps.GetSecret(bl.Location.SecretConfig, namespace)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
Even if one provides incorrect credentials to Object Store we pass this step and later fail during migration. Can you file a PWX to track this so a validation can be done here itself?

Reply:
https://portworx.atlassian.net/browse/PWX-32852